contrib/cmd/user-broker/user-broker.go

```diff
@@ -37,10 +42,39 @@ func init() {
 }
 
 func main() {
+	if err := run(); err != nil && err != context.Canceled && err != context.DeadlineExceeded {
+		glog.Fatalln(err)
```

Comment:
@arschles It should not always exit with 0 (like in your example), it should only exit with zero when err is `context.Canceled`, `context.DeadlineExceeded` or `http.ErrServerClosed`. We could `glog.Info()` them in an `else` clause, I don't mind that.